flexget/plugins/sites/passthepopcorn.py

```diff
@@ -288,8 +291,16 @@ def search(self, task, entry, config):
                     e['torrent_id'] = int(torrent['Id'])
                     e['golden_popcorn'] = torrent['GoldenPopcorn']
                     e['checked'] = torrent['Checked']
+                    e['scene'] = torrent['Scene']
                     e['uploaded_at'] = dateutil_parse(torrent['UploadTime'])
 
+                    e['ptp_remaster_title'] = torrent['RemasterTitle'] # tags such as remux, 4k remaster, etc.
+                    e['ptp_quality'] = torrent['Quality'] # high, ultra high, or standard definition
+                    e['ptp_resolution'] = torrent['Resolution'] # 1080p, 720p, etc.
+                    e['ptp_source'] = torrent['Source'] # blu-ray, dvd, etc.
+                    e['ptp_container'] = torrent['Container'] # mkv, vob ifo, etc.
+                    e['ptp_codec'] = torrent['Codec'] # x264, XviD, etc.
```

Comment:
Can we be certain that all these keys will exist in every response?

Reply:
Actually the only one we need to worry about is remastertitle which I missed the first time around. The rest are all require and will always be there. Regardless, I switched the key access to use dict.get() which should prevent any issues.